django_socio_grpc/tests/fakeapp/services/enum_service.py

```diff
@@ -0,0 +1,53 @@
+from typing import Annotated
+
+from django.db import models
+from fakeapp.grpc import fakeapp_pb2
+from fakeapp.models import EnumModel
+from fakeapp.serializers import (
+    EnumServiceAnnotatedSerializerSerializer,
+    EnumServiceSerializer,
+)
+
+from django_socio_grpc import generics
+from django_socio_grpc.decorators import grpc_action
+from django_socio_grpc.protobuf.generation_plugin import InMessageWrappedEnumGenerationPlugin
+
+
+class MyGRPCActionEnum(models.TextChoices):
```

Comment:
is it working with a simple python enum ? If TextChoices inherit from enum then it doesn't need more but if not we may want a test that check a simple enum is working

Reply:
Yes, actually wanted to put an enum here and forgot